src/ui/public/visualize/visualize.less

```diff
@@ -16,7 +16,7 @@ visualize {
     display: flex;
     flex-direction: row;
 
-    flex: 1 0;
+    flex: 1 1 0;
```

Comment:
This was still an issue in FF 49. The resolution ended up being `flex: 1 1 0` for all browsers except IE 11, which I used `flex: 1 0`